### PR TITLE
aegisub: 3.2.2 -> 3.3.2

### PR DESCRIPTION
**Pull request by somasis**
###### Description of changes

This also is a switch to wangqr's fork.
This fork is currently used by OpenSUSE, Fedora, Solus, and Void Linux. <https://repology.org/project/aegisub/versions>

<https://github.com/wangqr/aegisub>

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by AndersonTorres** [comment]
Why the cryptic version number?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/video/aegisub/default.nix [hunk 1]
```diff
@@ -37,64 +39,39 @@
...
 assert portaudioSupport -> (portaudio != null);
 
 let
+  luajit52 = luajit.override { enable52Compat = true; };
   inherit (lib) optional;
 in
 stdenv.mkDerivation rec {
   pname = "aegisub";
-  version = "3.2.2";
+  version = "3.3.2.20220513";
```
Comment (AndersonTorres) [inline pkgs/applications/video/aegisub/default.nix:58]: Why the cryptic version number?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/video/aegisub/default.nix (hunk 1 above)
Comment (somasis) [reply pkgs/applications/video/aegisub/default.nix]: I figured it made sense given that's the date of the Git commit being used. However, looking over my changes after reading this comment made me realize that using the latest Git actually was unnecessary; it's back to 3.3.2 (with the `no-git.patch` applied) now.

**Review by AndersonTorres — COMMENTED** [review]
LGTM, waiting for ofborg.

**Review by AndersonTorres — APPROVED** [review]

**Comment by somasis** [comment]
Is there anything holding this back from being merged?

**Comment by AndersonTorres** [comment]
Nothing to be honest. I was waiting for ofborg, but I forgot completely.

**Comment by somasis** [comment]
No problem. Thanks :)

**Pull request merged**